src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -454,6 +462,26 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     this.alreadyHandlingDeviceChange = false;
   }
 
+  private async handleDeviceStreamEnded(kind: string, deviceId: string): Promise<void> {
+    if (
+      kind === 'video' &&
+      this.boundAudioVideoController &&
+      this.boundAudioVideoController.videoTileController.hasStartedLocalVideoTile()
+    ) {
+      this.boundAudioVideoController.videoTileController.stopLocalVideoTile();
```

Comment:
can we have an additional method for this functionality to stop the LocalVideoTile so it can be reused in releaseMediaStream()

Reply:
We removed this in another place so it shows up in one place right now.